frontend/front/src/pages/Public/Components/CardHero.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Typography, Box, Grid, CardContent, CardActions } from "@mui/material";
+import ButtonGetPumnp from "./ButtonGetPump";
+
+const CardHero = ({ title, paragraphs, image }) => {
+  return (
+    <Grid
```

Comment:
So, I think the Grid here flex alignment is messing up the alignment when it's in the mobile version. There's a fix to this, but I don't remember. It has something to do with spacing it around or evenly. I'm going to left you find out how to fix it.

Reply:
I noticed it is a little messy. I will work on fixing it this week and create the mobile version.